changelogs/fragments/yaml_orderd_mappings.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+    - Use orderddict by defaul when importing mappings from YAML.
```

Comment:
```suggestion
    - Use OrderedDict by default when importing mappings from YAML.
```

Reply:
It looks like this was overlooked.

Reply:
i accepted, but then force pushed ... sorry...